src/app/shared/services/api/folder.repo.ts

```diff
@@ -40,8 +40,18 @@ export class FolderRepo extends BaseRepo {
   }
 
   public navigate(folderVO: FolderVO): Observable<FolderResponse> {
+    console.log(folderVO)
+    let response = {
+      ...folderVO
+    }
+    if(folderVO.type === 'type.folder.root.private'){
+      response.displayName = 'Private'
+    }
+
+    console.log(response)
+
```

Comment:
```suggestion
    const response = {
      ...folderVO
    };
    if (folderVO.type === 'type.folder.root.private') {
      response.displayName = 'Private';
    }

```
There are some leftover console.log statements here. Also, since `response` is never reassigned, let's go ahead and use `const` instead of `let` to define it.